internal-api/src/main/kotlin/com/example/internal/api/subject/OrdersNatsSubject.kt

```diff
@@ -0,0 +1,11 @@
+package com.example.internal.api.subject
+
+object OrdersNatsSubject {
```

Comment:
combine all subjects into one class, so the client of your API can see in one place what svc can do